tests/runfiles/common.run

```diff
@@ -898,3 +898,7 @@ tests = ['log_spacemap_import_logs']
 pre =
 post =
 tags = ['functional', 'log_spacemap']
+
+[tests/functional/zpool_influxdb]
+tests = 'zpool_influxdb'
```

Comment:
This must be a list:
```suggestion
tests = ['zpool_influxdb']
```

Strangely, the test runner was returning 0 after raising a TypeError exception about this, so though it appeared all the tests were passing, none of them actually ran. You can see in the details the zfstests step took only like 3 seconds.

Reply:
got it